### PR TITLE
chore: add twitter badge to readme

### PR DESCRIPTION
**Pull request by dhmlau**
Now that we're planning to post more development updates to Twitter, I'm thinking that it might be a good idea to add a twitter badge in readme.  The [nodejs.dev repo](https://github.com/nodejs/nodejs.dev#nodejsdev) and other popular repos have it too. 

Here is the preview from VSCode:

Screen Shot 2020-02-20 at 12 13 52 PM

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,7 @@
 [![Travis Build Status](https://travis-ci.com/strongloop/loopback-next.svg?branch=master)](https://travis-ci.com/strongloop/loopback-next)
 [![AppVeyor Build status](https://ci.appveyor.com/api/projects/status/q8vp7wrdn2ak6801/branch/master?svg=true)](https://ci.appveyor.com/project/strongloop/loopback-next/branch/master)
 [![Coverage Status](https://coveralls.io/repos/github/strongloop/loopback-next/badge.svg?branch=master)](https://coveralls.io/github/strongloop/loopback-next?branch=master)
+[![Twitter](https://img.shields.io/twitter/follow/strongloop.svg?style=social&label=Follow%20%40strongloop)](https://twitter.com/strongloop)
```
Comment (raymondfeng) [inline README.md]: Maybe have it on a separate line?

Comment (raymondfeng) [reply README.md]: What about LinkedIn/Facebook?

Comment (dhmlau) [reply README.md]: Let me look into the badge for LinkedIn/Facebook.

**Comment by dhmlau** [comment]
@raymondfeng, I'm looking into the branding guidelines from [Facebook](https://en.facebookbrand.com/facebookapp/) and [LinkedIn](https://brand.linkedin.com/policies). Here is the screen shot: 
Screen Shot 2020-02-24 at 11 40 00 AM

While looking for samples, I realized that it's not common for open source projects to put Facebook and LinkedIn as their badges. I'm good either way, but wanna let you know about this as well. Thanks.

**Pull request merged**